sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -204,23 +204,36 @@ class SentryPlugin implements Plugin<Project> {
 
                         def variantName = variant.buildType.name
                         def flavorName = variant.flavorName
+                        // When flavor is used in combination with dimensions, variant.flavorName will be a concatenation
+                        // of flavors of different dimensions
                         def propName = "sentry.properties"
-                        def possibleProps = [
-                                "${project.projectDir}/src/${variantName}/${propName}",
+                        // current flavor name takes priority
+                        def possibleProps = []
+                        variant.productFlavors.each {
+                            // flavors used with dimension come in second
+                            possibleProps.push("${project.projectDir}/src/${it.name}/${propName}")
+                        }
+
+                        possibleProps = [
                                 "${project.projectDir}/src/${flavorName}/${propName}",
+                                "${project.rootDir.toPath()}/src/${flavorName}/${propName}",
```

Comment:
Is there any reason why we are searching in `${project.rootDir.toPath()}/src/`? 
I think I have never seen an Android project with a `src` folder in its root 🤔

Reply:
I wanted to get rid of this but that'd could *possibly* be a breaking change. So I'd say we leave this for now and take into account on `2.0.0`.